com.unity.render-pipelines.high-definition/Editor/Material/Lit/ShaderGraph/LitPass.template

```diff
@@ -48,9 +48,6 @@ Pass
     $TransparentWritesMotionVec:        #define _WRITE_TRANSPARENT_MOTION_VECTOR 1
     $DepthOffset:                       #define _DEPTHOFFSET_ON 1
     $BlendMode.PreserveSpecular:        #define _BLENDMODE_PRESERVE_SPECULAR_LIGHTING 1
-    $NormalDropOffTS:                   #define _NORMAL_DROPOFF_TS 1
-    $NormalDropOffOS:                   #define _NORMAL_DROPOFF_OS 1
-    $NormalDropOffWS:                   #define _NORMAL_DROPOFF_WS 1
     $AttributesMesh.normalOS:           #define ATTRIBUTES_NEED_NORMAL
```

Comment:
Don't forget to also remove all the

        public static FieldDescriptor NormalDropOffTS =       new FieldDescriptor(string.Empty, "NormalDropOffTS", "_NORMAL_DROPOFF_TS 1");
        public static FieldDescriptor NormalDropOffOS =       new FieldDescriptor(string.Empty, "NormalDropOffOS", "_NORMAL_DROPOFF_OS 1");
        public static FieldDescriptor NormalDropOffWS =       new FieldDescriptor(string.Empty, "NormalDropOffWS", "_NORMAL_DROPOFF_WS 1");